protocol/0028-GOVE-governance.md

```diff
@@ -378,6 +378,12 @@ A proposal to close a market contains:
 
 Once market is closed the process cannot be reversed. Note that this implies that once a governance proposal to close the market has been voted in the market will definitely close at the enactment time of that vote at the latest. While the market is still open it's still possible to submit additional governance votes to close the market, however they'll only have any effect if their enactment date is prior to that of the market closure proposal which has already passed.
 
+If the market is in an auction of any type excluding the opening auction at the time the market closure governance proposal gets enacted, then the auction should uncross immediately, any trades resulting from it should be generated at the auction uncrossing price and then the system should proceed to close the market using the price (if applicable) provided by the proposal being enacted. If the market is in opening auction when the governance proposal to close it gets enacted then auction shouldn't uncross, market closes trivially as no trades have yet been generated.
+
+Attempting to enact the market closure governance proposal on a market in a `settled` [state](./0043-MKTL-market_lifecycle.md#market-status-descriptions) has no effect. When closing a market which needs the final price with a governance vote it's always the price supplied with the governance vote being enacted that gets used, even if the oracle price is available at that time.
```

Comment:
I'm assuming that the price has to be in the settlement decimal places but can you please clarify.

Reply:
Yes, please let me know if that's sufficient: https://github.com/vegaprotocol/specs/pull/1809/commits/8f820fe7f78868f5e0452f3541bfa019b255fc65

Reply:
I think it's fine if what you mean is actually the settlement asset's decimals.

Reply:
Ok, changed to market DP: https://github.com/vegaprotocol/specs/pull/1809/commits/6f3f31eaf316f23bc12b503cd4712aaf79a60441